app/api/resources/user.py

```diff
@@ -58,7 +60,9 @@ def get(cls):
         available_to_mentor. The current user's details are not returned.
         """
         user_id = get_jwt_identity()
-        return DAO.list_users(user_id, request.args.get("search", ""))
+        # Get `is_verified` boolean from request args
+        is_verified = True if request.args.get('is_verified') in TRUTHY else None
```

Comment:
Why can't is_verified be either a boolean parameter or even an integer, why are we using string matching with a hard coded list?

Why is is_verified being set to None and not False?

Reply:
For `GET /users/verified` api we were using is_verified = true. In `GET \users` api however, we are using None. So I have kept it same. 
I think None and False both will work same. I can test it and update you on the same.

Reply:
@ramitsawhney27 by default query parameter are treated as string. So we have to cast them in type first if we want something different than string. This casting is simple in case of other types but for boolean it is a bit tricky as different languages handles it differently.